docker-compose.override.yml

```diff
@@ -103,12 +103,20 @@ services:
     #restart: always
     ports:
      - 8074:80
-    networks: 
+    networks:
      - internal-net
      - external-net
     volumes:
      - /sessions
 
+  #########################################################
+  ## End-to-end tests #####################################
+  #########################################################

```

Comment:
Having the end-to-end tests implemented here will run the container whenever we do a `docker-compose up` - I am not sure we want to do this by default. Maybe it could be wise to define this in `docker-compose.test.yml` - but I am not sure whats the strategy here - would we developers need the container running to run fullstack tests?